omod/src/test/java/org/openmrs/module/aijar/page/controller/MarkPatientDeadTest.java

```diff
@@ -0,0 +1,95 @@
+package org.openmrs.module.aijar.page.controller;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Concept;
+import org.openmrs.Patient;
+import org.openmrs.api.context.Context;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+
+import java.util.Date;
+
+/**
+ * Created by lubwamasamuel on 19/15/2016.
+ */
+public class MarkPatientDeadTest extends BaseModuleWebContextSensitiveTest {
```

Comment:
Add a tests:
- Validation for future date of death
- What happens when a patient is already dead and they are being marked as alive